app/models/invenio_rdm_record_converter.rb

```diff
@@ -267,14 +266,13 @@ def rights(license_urls)
 
       if license_url == ALL_RIGHTS_RESERVED
         {
-          "rights": license_data[:"name"],
+          "title": license_data[:"name"]
```

Comment:
Since `id` is required, but it's hard to pick an `id` that is unique, what about adding

```
"id": "GALTER-ARR-1" 
```
It doesn't conflict with anything on the SPDX list and the `GALTER` prefix namespaces it enough. The `1` is just to futureproof against legalese change.

Reply:
Okay, did not realize it was required. I like the suggested 'id', and will add it.

Reply:
Oh what about link? Is that required?

Reply:
Never mind, I see that link is not required. Unless the documentation is not updated, just let me know and we can have a quick chat with the group about this.

Reply:
The docs are good on that one :sweat_smile: : `link` is not required. The best would be to not provide the key either in that case.

Reply:
@fenekku can I take your above comment as an approval of this PR then? `link` is omitted from "All Rights Reserved". No other case may have an "empty" link.